app/views/projects/new.html.slim

```diff
@@ -31,11 +31,12 @@ p.well Please make sure you have read our #{link_to "mentor guide", 'http://rail
       = f.input :source_url, as: :url, wrapper_html: { class: 'col-md-6' }, label: "Repository", placeholder: 'Link to your source code (Github, Gitlab, Bitbucket, …)' 
       = f.input :description, wrapper_html: { class: 'col-md-6' }, placeholder: "What's your project about?", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
       = f.input :issues_and_features, wrapper_html: { class: 'col-md-6' }, placeholder: "A list of possible features to work on, maybe even concrete Github issues", input_html: { rows: 5 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
+      = f.input :requirements, wrapper_html: { class: 'col-md-6' }, placeholder: "E.g. good knowledge of a specific programming language, proficiency with CSS, etc. Maybe a coding challenge (on Exercism.io, Codeschool, Codewars, etc.) that needs to be completed.", input_html: { rows: 6 }, hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
```

Comment:
I'm for some reason pro avoiding *"you"* in hints 🙈  - but I really like your point about
> something of your own choice

and
> skill level of programming language

However, I don't think it's necessary to repeat *"you can list requirements"* as it's already the title of the field.

How about something like:
> Skills students should ideally have - e.g. good knowledge of a specific programming language, proficiency with CSS, etc. Maybe illustrate via a coding challenge that needs to be completed (on Exercism.io, Codeschool, Codewars or somewhere else).

I deliberately would not put *"framework"* there, as it gets especially annoying in the JS world where they drop another one every 24hours and I feel forcing people to look at an new DSL for the same concepts all the time just puts additional pressure. Feel free to say I'm totally wrong.